tests/ArraydotifyTest.php

```diff
@@ -29,7 +29,7 @@ class ArraydotifyTest extends \PHPUnit\Framework\TestCase
         ]
     ];
 
-    public function setUp()
+    protected function setUp(): void
```

Comment:
To upgrade the PHPUnit version to `8`, it should change the PHPUnit fixture methods.